frontend/components/IntroductionVideo/IntroductionVideo.test.js

```diff
@@ -0,0 +1,12 @@
+import { render, screen } from "@testing-library/react";
+
+import IntroductionVideo from "./IntroductionVideo";
+
+describe("IntroductionVideo", () => {
```

Comment:
I would argue that IntroductionVideo doesn't really need testing, and that this test doesn't do much that's useful. It asserts that React renders the `<iframe />` with the expected test-id, but I feel it's okay to trust that React does what it's supposed to.

The only reason I can think of for testing something like this is if a prop given to the component, or some other side effect or internal component state, could cause the iframe not to be rendered.

For example, if the component took a `showVideo` component which determines if the iframe should appear, I think this is a good test to have

```js
function IntroductionVideo({ showVideo }) {
  render (
    {/* ... */}
    {showVideo && <iframe ... />}
    {/* ... */
  );
}
```

But in this case, React will always render the iframe.

It's up to you if you want to remove the test (it's not doing any harm), but hopefully this is useful for the future?

Reply:
Thanks, this is good to know. I'll leave it in for now as Pascal wrote it and like you said: it's not doing any harm.